packages/server/src/queue/queue.service.ts

```diff
@@ -1,9 +1,16 @@
+import {
```

Comment:
I'm alright not doing it now, but a big TODO should be queue service tests. The new ListQuestions stuff has like no tests afaik

Reply:
wasn't the whole thing we *didn't* know how to do this currently?